examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,205 @@
     "tags": []
```

Comment:
<p>Let's use a different function. This decodes empty sites correctly:</p><p> </p><pre class="ql-syntax" spellcheck="false">from collections import Counter

def get_counts(result):
"""Aggregate state counts from AHS shot results

A count of strings (of length = # of atoms) are returned, where
each character denotes the state of an atom (site):
    e: empty site
    r: Rydberg state spin
    g: ground state spin
  
Args:
    result (braket.tasks.analog_hamiltonian_simulation_quantum_task_result.AnalogHamiltonianSimulationQuantumTaskResult)

Returns
    dict: number of times each state configuration is measured

"""
    state_counts = Counter()
    states = ['e', 'r', 'g']
    for shot in result.measurements:
    pre = shot.pre_sequence
    post = shot.post_sequence
    state_idx = np.array(pre) * (1 + np.array(post))
    state = "".join(map(lambda s_idx: states[s_idx], state_idx))
    state_counts.update((state,))
    return dict(state_counts)
    
counts = get_counts(result)
    
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>